### ISSUE TITLE
[Camera Support]: Timeout/EOF on a wired POE intercom

### ISSUE THREAD

**Issue by xavan83** [blakeblackshear/frigate#8882]
### Describe the problem you are having

I receive a lot (6-7 times per hour) an error on this camera: 
>192.168.0.70:554: i/o timeout" url=rtsp:/192.168.0.70:554/Streaming/Channels/101

This is a Hikvision poe powered ds-kb8113-ime intercom located 100meters away from my home.
Maybe the network connection is not perfect but I am unable to do a better one
I am using it using the main stream (fullhd + audio) but the same problem occurs using the sub stream.
How can I reduce the timeout/buffer/reorder frame ?

### Version

0.13.0-614A36A (beta6)

### Frigate config file

```yaml
go2rtc:
  streams:
    sonnette:
      - rtsp://192.168.0.70:554/Streaming/Channels/101
cameras:
  sonnette:
    ffmpeg:
      output_args:
        record: preset-record-generic-audio-aac
      inputs:
        - path: rtsp://127.0.0.1:8554/sonnette
          roles:
            - detect,record
    detect:
      fps: 5
      enabled: True
      width: 1920
      height: 1080
    motion:
      mask:
        - 0,674,880,739,1514,672,1920,597,1920,0,0,0
    mqtt:
      enabled: True
      timestamp: True
      bounding_box: True
      crop: False
      quality: 100
      required_zones: []
    objects:
      filters:
        person:
           min_score: 0.3
           threshold: 0.5
           min_area: 10000
        cars:
          min_score: 0.3
          threshold: 0.5
      track:
        - person
        - bicycle
        - car
        - motorcycle
        - bus
        - cat
        - dog
        - horse
```

### Relevant log output

```shell
2023-12-07 12:17:22.070139725  12:17:22.070 WRN github.com/AlexxIT/go2rtc/internal/streams/producer.go:171 > error="read tcp 172.18.0.2:50360->192.168.0.70:554: i/o timeout" url=rtsp://192.168.0.70:554/Streaming/Channels/101
```

### FFprobe output from your camera

```shell
ffprobe version 6.0-6ubuntu1 Copyright (c) 2007-2023 the FFmpeg developers
  built with gcc 13 (Ubuntu 13.2.0-2ubuntu1)
  configuration: --prefix=/usr --extra-version=6ubuntu1 --toolchain=hardened --libdir=/usr/lib/x86_64-linux-gnu --incdir=/usr/include/x86_64-linux-gnu --arch=amd64 --enable-gpl --disable-stripping --enable-gnutls --enable-ladspa --enable-libaom --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-libcdio --enable-libcodec2 --enable-libdav1d --enable-libflite --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libglslang --enable-libgme --enable-libgsm --enable-libjack --enable-libmp3lame --enable-libmysofa --enable-libopenjpeg --enable-libopenmpt --enable-libopus --enable-libpulse --enable-librabbitmq --enable-librist --enable-librubberband --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libsrt --enable-libssh --enable-libsvtav1 --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx265 --enable-libxml2 --enable-libxvid --enable-libzimg --enable-libzmq --enable-libzvbi --enable-lv2 --enable-omx --enable-openal --enable-opencl --enable-opengl --enable-sdl2 --disable-sndio --enable-libjxl --enable-pocketsphinx --enable-librsvg --enable-libvpl --disable-libmfx --enable-libdc1394 --enable-libdrm --enable-libiec61883 --enable-chromaprint --enable-frei0r --enable-libx264 --enable-libplacebo --enable-librav1e --enable-shared
  libavutil      58.  2.100 / 58.  2.100
  libavcodec     60.  3.100 / 60.  3.100
  libavformat    60.  3.100 / 60.  3.100
  libavdevice    60.  1.100 / 60.  1.100
  libavfilter     9.  3.100 /  9.  3.100
  libswscale      7.  1.100 /  7.  1.100
  libswresample   4. 10.100 /  4. 10.100
  libpostproc    57.  1.100 / 57.  1.100
[rtsp @ 0x557e0ce37c40] Missing PPS in sprop-parameter-sets, ignoring
Input #0, rtsp, from 'rtsp://192.168.0.70:554/Streaming/Channels/101':
  Metadata:
    title           : Media Presentation
  Duration: N/A, start: 0.040000, bitrate: N/A
  Stream #0:0: Video: h264 (Baseline), yuvj420p(pc, bt709, progressive), 1920x1080, 12 fps, 12 tbr, 90k tbn
  Stream #0:1: Audio: pcm_mulaw, 8000 Hz, mono, s16, 64 kb/s
```

### Operating system

Other Linux

### Install method

Docker Compose

### Coral version

CPU (no coral)

### Network connection

Wired

### Camera make and model

Hikvision

**Comment by xavan83** [comment]
Timeout still occurs in go2rtc log:
2023-12-07 16:54:09.790488770  16:54:09.790 WRN github.com/AlexxIT/go2rtc/internal/streams/producer.go:171 > error="read tcp 172.18.0.2:33334->192.168.0.70:554: i/o timeout" url=rtsp://admin:VERBATIM83@192.168.0.70:554/Streaming/Channels/101#timeout=10

At the same time I found these log entries:
2023-12-07 16:48:30.728377895  [2023-12-07 16:48:30] frigate.record.maintainer      WARNING : Unable to keep up with recording segments in cache for jardin. Keeping the 6 most recent segments out of 7 and discarding the rest...
2023-12-07 16:48:35.729526196  [2023-12-07 16:48:35] frigate.record.maintainer      WARNING : Unable to keep up with recording segments in cache for jardin. Keeping the 6 most recent segments out of 9 and discarding the rest...
2023-12-07 16:48:40.716093444  [2023-12-07 16:48:40] frigate.record.maintainer      WARNING : Unable to keep up with recording segments in cache for jardin. Keeping the 6 most recent segments out of 7 and discarding the rest...
2023-12-07 16:50:55.924908050  [2023-12-07 16:50:55] frigate.record.maintainer      DEBUG   : Copied /media/frigate/recordings/2023-12-07/15/sonnette/50.41.mp4 in 0.12321591377258301 seconds.
2023-12-07 16:51:05.933575649  [2023-12-07 16:51:05] frigate.record.maintainer      DEBUG   : Copied /media/frigate/recordings/2023-12-07/15/sonnette/50.51.mp4 in 0.08966302871704102 seconds.
2023-12-07 16:51:15.911386007  [2023-12-07 16:51:15] frigate.record.maintainer      DEBUG   : Copied /media/frigate/recordings/2023-12-07/15/sonnette/51.01.mp4 in 0.1435561180114746 seconds.
2023-12-07 16:53:55.912763185  [2023-12-07 16:53:55] frigate.record.maintainer      DEBUG   : Copied /media/frigate/recordings/2023-12-07/15/sonnette/53.41.mp4 in 0.10363507270812988 seconds.
2023-12-07 16:54:20.738044156  [2023-12-07 16:54:20] frigate.record.maintainer      DEBUG   : Found 1 old frames.
2023-12-07 16:54:20.879548078  [2023-12-07 16:54:20] frigate.record.maintainer      DEBUG   : Copied /media/frigate/recordings/2023-12-07/15/sonnette/53.51.mp4 in 0.06303095817565918 seconds.
2023-12-07 16:54:40.726883095  [2023-12-07 16:54:40] watchdog.sonnette              INFO    : No frames received from sonnette in 20 seconds. Exiting ffmpeg...
2023-12-07 16:54:40.728100158  [2023-12-07 16:54:40] watchdog.sonnette              INFO    : Waiting for ffmpeg to exit gracefully...
2023-12-07 16:54:40.799873623  [2023-12-07 16:54:40] frigate.video                  ERROR   : sonnette: Unable to read frames from ffmpeg process.
2023-12-07 16:54:40.801392076  [2023-12-07 16:54:40] frigate.video                  ERROR   : sonnette: Unable to read frames from ffmpeg process.
2023-12-07 16:54:40.801398068  [2023-12-07 16:54:40] frigate.video                  ERROR   : sonnette: Unable to read frames from ffmpeg process.
2023-12-07 16:54:40.801399685  [2023-12-07 16:54:40] frigate.video                  ERROR   : sonnette: Unable to read frames from ffmpeg process.
2023-12-07 16:54:40.802442320  [2023-12-07 16:54:40] frigate.video                  ERROR   : sonnette: Unable to read frames from ffmpeg process.
2023-12-07 16:54:40.802447558  [2023-12-07 16:54:40] frigate.video                  ERROR   : sonnette: Unable to read frames from ffmpeg process.
2023-12-07 16:54:40.802449182  [2023-12-07 16:54:40] frigate.video                  ERROR   : sonnette: ffmpeg process is not running. exiting capture thread...
2023-12-07 16:54:45.733478425  [2023-12-07 16:54:45] frigate.record.maintainer      DEBUG   : Found 1 old frames.
2023-12-07 16:54:50.866117715  [2023-12-07 16:54:50] watchdog.sonnette              ERROR   : Ffmpeg process crashed unexpectedly for sonnette.
2023-12-07 16:54:50.866638529  [2023-12-07 16:54:50] watchdog.sonnette              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-12-07 16:54:50.866641658  [2023-12-07 16:54:50] ffmpeg.sonnette.detect         ERROR   : [rtsp @ 0x55bded7832c0] Missing PPS in sprop-parameter-sets, ignoring
2023-12-07 16:54:50.866647642  [2023-12-07 16:54:50] ffmpeg.sonnette.detect         ERROR   : [segment @ 0x55bded798bc0] Timestamps are unset in a packet for stream 0. This is deprecated and will stop working in the future. Fix your code to set the timestamps properly
2023-12-07 16:54:50.866649597  [2023-12-07 16:54:50] ffmpeg.sonnette.detect         ERROR   : [h264 @ 0x55bded8925c0] Using possibly-incompatible profile 100 instead.
2023-12-07 16:54:50.866652168  [2023-12-07 16:54:50] ffmpeg.sonnette.detect         ERROR   :     Last message repeated 1 times
2023-12-07 16:54:50.866693317  [2023-12-07 16:54:50] ffmpeg.sonnette.detect         ERROR   : [segment @ 0x55bded798bc0] Non-monotonous DTS in output stream 0:0; previous: 0, current: 0; changing to 1. This may result in incorrect timestamps in the output file.
2023-12-07 16:54:50.866697095  [2023-12-07 16:54:50] ffmpeg.sonnette.detect         ERROR   : [h264 @ 0x55bded7fe680] data partitioning is not implemented. Update your FFmpeg version to the newest one from Git. If the problem still occurs, it means that your file has a feature which has not been implemented.
2023-12-07 16:54:50.866716011  [2023-12-07 16:54:50] ffmpeg.sonnette.detect         ERROR   : [h264 @ 0x55bded7fe680] If you want to help, upload a sample of this file to https://streams.videolan.org/upload/ and contact the ffmpeg-devel mailing list. (ffmpeg-devel@ffmpeg.org)
2023-12-07 16:54:50.866718053  [2023-12-07 16:54:50] ffmpeg.sonnette.detect         ERROR   : [h264 @ 0x55bded8925c0] Increasing reorder buffer to 10
2023-12-07 16:54:50.867364254  [2023-12-07 16:54:50] ffmpeg.sonnette.detect         ERROR   : rtsp://127.0.0.1:8554/sonnette: Connection timed out
2023-12-07 16:54:50.867369288  [2023-12-07 16:54:50] ffmpeg.sonnette.detect         ERROR   :     Last message repeated 1 times
2023-12-07 16:54:50.867371186  [2023-12-07 16:54:50] ffmpeg.sonnette.detect         ERROR   : [rtsp @ 0x55bded7832c0] RTP: PT=60: bad cseq 388e expected=2cac
